pkg/cli/tools/binary_tools.go

```diff
@@ -17,21 +17,17 @@ limitations under the License.
 package tools
 
 import (
-	"context"
 	"fmt"
+	"io"
+	"net/http"
 	"os"
 	"path"
 	"runtime"
-
-	"github.com/radius-project/radius/pkg/version"
-	"oras.land/oras-go/v2"
-	"oras.land/oras-go/v2/content/file"
-	"oras.land/oras-go/v2/registry/remote"
 )
 
 const (
 	// binaryRepo is the name of the remote bicep binary repository
-	binaryRepo = "ghcr.io/radius-project/radius/bicep/rad-bicep/"
+	binaryRepo = "https://github.com/Azure/bicep/releases/latest/download/"
```

Comment:
Would getting the `latest` version all the time cause a problem? Could there be a breaking change that could affect us?

Reply:
I think we'd want to always get the latest version of bicep. It is possible that there are breaking changes since this is now a dependency instead of something we own. But we will have dependency updates and testing as part of the merge (this will likely come in closer to the end) that will notify us if there's a breaking change.